src/coreclr/jit/emit.h

```diff
@@ -1547,11 +1547,34 @@ class emitter
         {
             return _idEvexbContext != 0;
         }
-        void idSetEvexbContext()
+
+        void idSetEvexbContext(insOpts instOptions)
+        {
+            if (instOptions == INS_OPTS_EVEX_b)
+            {
+                _idEvexbContext = 1;
+            }
+            else if (instOptions == INS_OPTS_EVEX_er_rd)
+            {
+                _idEvexbContext = 1;
+            }
```

Comment:
Is it worth aliasing these options to remove a branch? That is, make `INS_OPTS_EVEX_b` and `INS_OPTS_EVEX_er_rd` be the "same"?

Which it is ultimately comes down to whether a memory operand is contained